app/components/publications/batch-documents-publication-modal.hbs

```diff
@@ -0,0 +1,51 @@
+<WebComponents::AuModal>
+  <WebComponents::AuModal::Header @title={{ t "documents" }} @closeModal={{this.onCancel}} />
+  <WebComponents::AuModal::Body>
+    {{!-- template-lint-disable table-groups --}}
+    <table class="auk-table auk-table--styled auk-table--bordered">
+      <thead>
+        <tr>
+          <th>{{t "name"}}</th>
+          <th>{{t "document-type"}}</th>
+          <th></th>
+        </tr>
+      </thead>
+
+      <Documents::DocumentList
+        @pieces={{this.pieces}}
+        {{!-- should be tbody tag; workaround for Documents::DocumentList rendering a div --}}
+        style="display: table-row-group;"
+        as |pieces|>
+        {{#each pieces as |piece|}}
+          <tr>
+            <td>
+              {{piece.name}}
+            </td>
+            <td>
+              {{piece.file.extension}}
+            </td>
```

Comment:
Dit is de kolom van document-type (nota, decreet, BVR), maar de data dat erin komt is het type van bestand (excel, word, pdf)
```
{{piece.documentContainer.type}}
```